src/actions/actions/services/dispatchers/UpdateServiceNotifications.js

```diff
@@ -0,0 +1,26 @@
+import {
+    ERROR_SERVICE_ACCOUNT,
+    SET_SERVICE_ACCOUNT
+  } from "../../../../utils/constants/storeType";
+  import { VERUSID } from "../../../../utils/constants/intervalConstants";
+  import { updateServiceDataValue } from "./UpdateServiceDataValue";
+
+  import { checkVerusIdNotificationsForUpdates } from "./verusid/verusid";
+  
+  const fetchChannels = () => {
+    return {
+      [VERUSID]: (channelStore) =>
+        checkVerusIdNotificationsForUpdates(channelStore),
+    };
+  };
+  
+  export const updateServiceNotifications = (state, dispatch, channels) =>
+    updateServiceDataValue(
+      state,
+      dispatch,
+      channels,
+      SET_SERVICE_ACCOUNT,
```

Comment:
This should work such that instead of dispatching SET_SERVICE_ACCOUNT, it dispatches a SET_SERVICE_NOTIFICATIONS action that follows the structure of all the other updateService<x> functions (action with {type: TYPE, payload: {channel: channel_name, body: {}}}) by making the updateServiceDataValue return {channel: channel_name, body: {}}. Its fine if the body is empty, as the function ultimately serves to dispatch notifications, but it should be a trackable redux event with a type. Also this means creating a ERROR_SERVICE_NOTIFICATIONS type and handling that in the error reducer so that errors with the notification function get stored in the redux store as well.